src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -197,7 +197,7 @@ public string[] ReferencedFiles
         /// </summary>
         /// <param name="text">Input string to be split up into lines.</param>
         /// <returns>The lines in the string.</returns>
-        public static IList<string> GetLines(string text)
+        public static List<string> GetLines(string text)
```

Comment:
Oof that's an odd one.  Kind of seems like that shouldn't have been public.

I think it's a pretty safe bet that no one is using it... I would have never thought to look for a static get lines method on that class.  That said, you could make an alternate private `GetLinesInternal` that is typed as `List<string>` then keep this method the same and have it implicitly cast to `IList<string>`

Either way, might consider decorating with `Obsolete`.  That should probably either be moved to a utility class or just made private.

Reply:
I get the impression that there are quite a few APIs exposed as public that really should be internal PSES, which is something I would ideally like to address with v2. It's a bit tricky because the way the project is broken up means some things need to be shared between libraries, but on the other hand those things are opportunities to design a real module API. But yeah, a lot of our public APIs I think need to be retired or changed.